packages/react/src/menu-content/MenuContent.tsx

```diff
@@ -0,0 +1,23 @@
+import * as RadixMenu from "@radix-ui/react-dropdown-menu";
+import { forwardRef } from "react";
+
+import { Box, type BoxProps } from "../box";
+import * as styles from "./MenuContent.css";
+
+type MenuContentProps = BoxProps<typeof RadixMenu.Content>;
```

Comment:
```suggestion
type MenuContentProps = BoxProps<"div", ComponentPropsWithRef<typeof RadixMenu.Content>>;
```

Reply:
why do we need to change this? what's there right now looks good to me

Reply:
If we don't need to extend props, should we still use the BoxProps vai? Else 
` type MenuContentProps=  ComponentPropsWithRef< typeof  RadixMenu.Content>` isn't fine enough vai?

Reply:
we still need `BoxProps` to accept sprinkles

Reply:
Got it vai,  then as we are open to accept sprinkles in almost all components even if we don't need to extend any props, we should also use it for only a component-props.